stacks/ActivitiesStack.tsx

```diff
@@ -162,9 +176,17 @@ export function TabNavigation () {
       <Tab.Screen name="Notifications" component={TestNotifScreen} />
       <Tab.Screen
         name="Profile"
-        component={ProfileScreenNavWrapper}
-        initialParams={{user: UserMocks.Mia}}
-      />
+        options={({route}) => ({
+            tabBarVisible: getTabBarVisibility(route)
+          })
+        }
+      >
+        {() => 
+          <ProfileStack.Navigator>
+            {profileScreens}
+          </ProfileStack.Navigator>
+        }
```

Comment:
Can't we just have a component which holds onto a profile stack navigator instead of this?